app/models/call_for_evidence_reminder.rb

```diff
@@ -0,0 +1,24 @@
+class CallForEvidenceReminder
+  WEEKS_FROM_CLOSE_TO_REMINDER = 8
+
+  class << self
+    def send_reminder
+      CallForEvidence.awaiting_response.closed_at_or_within_24_hours_of(WEEKS_FROM_CLOSE_TO_REMINDER.weeks.ago).each do |call_for_evidence|
+        log(call_for_evidence)
+        MultiNotifications.call_for_evidence_reminder(call_for_evidence).map(&:deliver_now)
+      end
+    end
+
+  private
+
+    def log(call_for_evidence)
+      Rails.logger.info("Sending reminder for call for evidence ##{call_for_evidence.id} '#{call_for_evidence.title}' to #{obfuscated_email_addresses(call_for_evidence)}")
+    end
+
+    def obfuscated_email_addresses(call_for_evidence)
```

Comment:
This doesn't really feel to me like something the reminder model should be responsible for, but given the Consultation reminder also is I think a separate PR would be appropriate if we want to invest in separating that responsibility